susemanager-utils/susemanager-sls/salt/services/kiwi-image-server.sls

```diff
@@ -1,6 +1,6 @@
 # Image Server installation state - part of SUSE Manager for Retail
 #
-# Copyright © 2017, 2018 SUSE LLC
+# Copyright (c) 2017, 2018 SUSE LLC
```

Comment:
Can you also update a year when you are at it? And also update year in all here modified files?